x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.tsx

```diff
@@ -383,6 +383,7 @@ export const BuilderEntryItem: React.FC<EntryItemProps> = ({
             isClearable={false}
             onChange={handleFieldListValueChange}
             data-test-subj="exceptionBuilderEntryFieldList"
+            allowLargeValueLists={allowLargeValueLists}
```

Comment:
@dplumlee  Tested locally and saw the following:

1. I created a Text type value list with name `text.txt`
2. Created a New Terms rule
3. Went to `Rule exceptions` and attempted to add that value list as an exception with the `is in list` operator.
4. I was expecting the `text.txt` list not to show in the dropdown, or an error being thrown if I tried to add that exception, but I could add it:
![image](https://user-images.githubusercontent.com/5354282/189391105-47131723-e6df-4187-92c0-d15ba40d1ceb.png)
![image](https://user-images.githubusercontent.com/5354282/189390976-b3e30ec5-6d81-4721-9da6-20837fed6290.png)

Is this behaviour expected?

Reply:
nope, forgot to add that logic when we dropped `text` lists from the small list category, good find